src/styles/blocks/card.scss

```diff
@@ -0,0 +1,75 @@
+.card {
+  width: 199px;
+  height: 100%;
+  border: 1px solid rgb(240, 7, 228);
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(/src/images/imac.jpeg);
+    background-size: cover;
+    width: 160px;
+    height: 134px;
+    margin-top: 33px;
+    margin-left: 20px;
```

Comment:
Hardcoded pixel values for margins can cause issues with responsiveness. Consider using relative units like percentages or use flexbox centering properties to center the image within the card.